configs/fcaf3d/README.md

```diff
@@ -0,0 +1,45 @@
+# FCAF3D: Fully Convolutional Anchor-Free 3D Object Detection
+
+> [FCAF3D: Fully Convolutional Anchor-Free 3D Object Detection](https://arxiv.org/abs/2112.00322)
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+Recently, promising applications in robotics and augmented reality have attracted considerable attention to 3D object detection from point clouds. In this paper, we present FCAF3D --- a first-in-class fully convolutional anchor-free indoor 3D object detection method. It is a simple yet effective method that uses a voxel representation of a point cloud and processes voxels with sparse convolutions. FCAF3D can handle large-scale scenes with minimal runtime through a single fully convolutional feed-forward pass. Existing 3D object detection methods make prior assumptions on the geometry of objects, and we argue that it limits their generalization ability. To eliminate prior assumptions, we propose a novel parametrization of oriented bounding boxes that allows obtaining better results in a purely data-driven way. The proposed method achieves state-of-the-art 3D object detection results in terms of mAP@0.5 on ScanNet V2 (+4.5), SUN RGB-D (+3.5), and S3DIS (+20.5) datasets.
+
+<div align="center">
+<img src="https://user-images.githubusercontent.com/6030962/182842796-98c10576-d39c-4c2b-a15a-a04c9870919c.png" width="800"/>
+</div>
+
+## Introduction
+
+We implement FCAF3D and provide the result and checkpoints on the ScanNet dataset.
+`Max` and `mean` metrics are copied from the paper and `ours` is for provided checkpoint.
+`Mean` value is averaged across 5 train runs followed by 5 test runs.
```

Comment:
we train each setting 5 times and test each training trial 5 times, then the average performance of these 25 trials is reported to account for algorithm randomness.